### ISSUE TITLE
A question about memloss

### ISSUE THREAD

**Issue by zxz0928** [maple-research-lab/AdCo#12]
logits=torch.softmax(logits,dim=1)
    batch_prob=torch.sum(logits[:,:logits.size(0)],dim=1)
    batch_prob=torch.mean(batch_prob)
    mem_losses.update(batch_prob.item(),logits.size(0))
why calculate mem_losses use the method above rather than cross-entropy loss?
Thank you very much.

**Comment by wang3702** [comment]
That's is not mem loss, we just use that to track the sum of in-batch probabilities.

**Comment by zxz0928** [comment]
Thank you for your reply.But I still can't understand why we need to track the sum of in-batch probabilities rather than loss?and how the in-batch probabilities value changes means that the experimental result is correct.

**Comment by wang3702** [comment]
That can reflect how much the memory bank negatives contribute to the loss optimization.
